Subject: [PATCH] cleaning frameworks
---
Index: CefGlue.Demo.WPF/App.xaml.cs
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/CefGlue.Demo.WPF/App.xaml.cs b/CefGlue.Demo.WPF/App.xaml.cs
--- a/CefGlue.Demo.WPF/App.xaml.cs	(revision ef20aa2d39eaa29b410300ff2ab2b5e7cef4593a)
+++ b/CefGlue.Demo.WPF/App.xaml.cs	(date 1675327096489)
@@ -1,8 +1,3 @@
-using System;
-using System.Collections.Generic;
-using System.Configuration;
-using System.Data;
-using System.Linq;
 using System.Windows;
 
 namespace Xilium.CefGlue.Demo.WPF
Index: CefGlue.Common/CefGlue.Common.csproj
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/CefGlue.Common/CefGlue.Common.csproj b/CefGlue.Common/CefGlue.Common.csproj
--- a/CefGlue.Common/CefGlue.Common.csproj	(revision ef20aa2d39eaa29b410300ff2ab2b5e7cef4593a)
+++ b/CefGlue.Common/CefGlue.Common.csproj	(date 1675274928867)
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFrameworks>net472;$(DotnetStandardVersion)</TargetFrameworks>
+        <TargetFramework>$(DotnetStandardVersion)</TargetFramework>
         <AssemblyName>Xilium.CefGlue.Common</AssemblyName>
         <RootNamespace>Xilium.CefGlue.Common</RootNamespace>
         <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
Index: CefGlue.Demo.WPF/CefGlue.Demo.WPF.csproj
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/CefGlue.Demo.WPF/CefGlue.Demo.WPF.csproj b/CefGlue.Demo.WPF/CefGlue.Demo.WPF.csproj
--- a/CefGlue.Demo.WPF/CefGlue.Demo.WPF.csproj	(revision ef20aa2d39eaa29b410300ff2ab2b5e7cef4593a)
+++ b/CefGlue.Demo.WPF/CefGlue.Demo.WPF.csproj	(date 1675333207628)
@@ -77,18 +77,6 @@
     <DefineConstants>DEBUG;TRACE</DefineConstants>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="System" />
-    <Reference Include="System.Data" />
-    <Reference Include="System.Xml" />
-    <Reference Include="System.Core" />
-    <Reference Include="System.Xml.Linq" />
-    <Reference Include="System.Data.DataSetExtensions" />
-    <Reference Include="WindowsBase" />
-    <Reference Include="PresentationCore" />
-    <Reference Include="PresentationFramework" />
-    <Reference Include="System.Xaml" Condition="$(TargetFrameworkVersion)!='v3.5'" />
-  </ItemGroup>
-  <ItemGroup>
     <ApplicationDefinition Include="App.xaml">
       <Generator>MSBuild:Compile</Generator>
       <SubType>Designer</SubType>
@@ -144,36 +132,44 @@
     </None>
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\CefGlue.BrowserProcess\CefGlue.BrowserProcess.csproj">
-      <Project>{1e60fb67-049e-4877-92a9-d5480f81a324}</Project>
-      <Name>CefGlue.BrowserProcess</Name>
-    </ProjectReference>
-    <ProjectReference Include="..\CefGlue.Common.Shared\CefGlue.Common.Shared.csproj">
-      <Project>{BA0F8218-A7D3-4C90-AA4D-F914A3C974E9}</Project>
-      <Name>CefGlue.Common.Shared</Name>
-    </ProjectReference>
-    <ProjectReference Include="..\CefGlue.Common\CefGlue.Common.csproj">
-      <Project>{F8F4D0B8-5777-41A1-AB3C-0E324E61FAF9}</Project>
-      <Name>CefGlue.Common</Name>
-    </ProjectReference>
-    <ProjectReference Include="..\CefGlue.WPF\CefGlue.WPF.csproj">
-      <Project>{447c6790-e36e-4a54-bfac-06c92fff4aa3}</Project>
-      <Name>CefGlue.WPF</Name>
-    </ProjectReference>
-    <ProjectReference Include="..\CefGlue\CefGlue.csproj">
-      <Project>{703FF300-34B9-48D4-A5D6-E741F48D5F95}</Project>
-      <Name>CefGlue</Name>
-    </ProjectReference>
+    <ProjectReference Include="..\CefGlue.Common.Shared\CefGlue.Common.Shared.csproj" />
+    <ProjectReference Include="..\CefGlue.Common\CefGlue.Common.csproj" />
+    <ProjectReference Include="..\CefGlue.WPF\CefGlue.WPF.csproj" />
+    <ProjectReference Include="..\CefGlue\CefGlue.csproj" />
+  </ItemGroup>
+  <ItemGroup>
+    <Reference Include="PresentationCore" />
+    <Reference Include="PresentationFramework" />
+    <Reference Include="System" />
+    <Reference Include="System.XAML" />
+    <Reference Include="WindowsBase" />
+    <Reference Include="System.CodeDom" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="System.Runtime.WindowsRuntime.UI.Xaml">
+      <Version>4.7.0</Version>
+    </PackageReference>
+  </ItemGroup>
+  <ItemGroup>
+    <BootstrapperPackage Include=".NETFramework,Version=v4.7.2">
+      <Visible>False</Visible>
+      <ProductName>Microsoft .NET Framework 4.7.2 %28x86 and x64%29</ProductName>
+      <Install>true</Install>
+    </BootstrapperPackage>
+    <BootstrapperPackage Include="Microsoft.Net.Framework.3.5.SP1">
+      <Visible>False</Visible>
+      <ProductName>.NET Framework 3.5 SP1</ProductName>
+      <Install>false</Install>
+    </BootstrapperPackage>
   </ItemGroup>
-  <ItemGroup />
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <Import Project="$(SolutionDir)\CefGlue.CopyLocal.props" />
   <Import Project="$(SolutionDir)\CefGlue.Common\build\CefGlue.Common.targets" />
   <!-- To modify your build process, add your task inside one of the targets below and uncomment it. 
-       Other similar extension points exist, see Microsoft.Common.targets.
-  <Target Name="BeforeBuild">
-  </Target>
-  <Target Name="AfterBuild">
-  </Target>
-  -->
+         Other similar extension points exist, see Microsoft.Common.targets.
+    <Target Name="BeforeBuild">
+    </Target>
+    <Target Name="AfterBuild">
+    </Target>
+    -->
 </Project>
\ No newline at end of file
Index: CefGlue/CefGlue.csproj
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/CefGlue/CefGlue.csproj b/CefGlue/CefGlue.csproj
--- a/CefGlue/CefGlue.csproj	(revision ef20aa2d39eaa29b410300ff2ab2b5e7cef4593a)
+++ b/CefGlue/CefGlue.csproj	(date 1675274590503)
@@ -1,10 +1,11 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net472;$(DotnetStandardVersion)</TargetFrameworks>
+    <TargetFramework>$(DotnetStandardVersion)</TargetFramework>
     <AssemblyName>Xilium.CefGlue</AssemblyName>
     <Configurations>Debug;Release;Debug_WindowlessRender</Configurations>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <LangVersion>latest</LangVersion>
   </PropertyGroup>
 
   <ItemGroup>
Index: CefGlue.Demo.WPF/BrowserView.xaml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/CefGlue.Demo.WPF/BrowserView.xaml b/CefGlue.Demo.WPF/BrowserView.xaml
--- a/CefGlue.Demo.WPF/BrowserView.xaml	(revision ef20aa2d39eaa29b410300ff2ab2b5e7cef4593a)
+++ b/CefGlue.Demo.WPF/BrowserView.xaml	(date 1675328555174)
@@ -1,7 +1,7 @@
-<UserControl xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-        x:Class="Xilium.CefGlue.Demo.WPF.BrowserView"
-        xmlns:wpf="clr-namespace:Xilium.CefGlue.WPF;assembly=Xilium.CefGlue.WPF">
+<UserControl  x:Class="Xilium.CefGlue.Demo.WPF.BrowserView"
+              xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+              xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+              xmlns:wpf="clr-namespace:Xilium.CefGlue.WPF;assembly=Xilium.CefGlue.WPF">
 
     <DockPanel LastChildFill="True">
         <TextBox x:Name="addressTextBox" DockPanel.Dock="Top" KeyDown="OnAddressTextBoxKeyDown" />
Index: CefGlue.Common.Shared/CefGlue.Common.Shared.csproj
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/CefGlue.Common.Shared/CefGlue.Common.Shared.csproj b/CefGlue.Common.Shared/CefGlue.Common.Shared.csproj
--- a/CefGlue.Common.Shared/CefGlue.Common.Shared.csproj	(revision ef20aa2d39eaa29b410300ff2ab2b5e7cef4593a)
+++ b/CefGlue.Common.Shared/CefGlue.Common.Shared.csproj	(date 1675274592845)
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net472;$(DotnetStandardVersion)</TargetFrameworks>
+    <TargetFramework>$(DotnetStandardVersion)</TargetFramework>
     <AssemblyName>Xilium.CefGlue.Common.Shared</AssemblyName>
     <RootNamespace>Xilium.CefGlue.Common.Shared</RootNamespace>
     <Configurations>Debug;Release;Debug_WindowlessRender</Configurations>
Index: CefGlue.Tests/CefGlue.Tests.csproj
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/CefGlue.Tests/CefGlue.Tests.csproj b/CefGlue.Tests/CefGlue.Tests.csproj
--- a/CefGlue.Tests/CefGlue.Tests.csproj	(revision ef20aa2d39eaa29b410300ff2ab2b5e7cef4593a)
+++ b/CefGlue.Tests/CefGlue.Tests.csproj	(date 1675274573850)
@@ -1,7 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>$(DotnetVersion)</TargetFramework>
     <IsPackable>false</IsPackable>
     <Configurations>Debug;Release;Debug_WindowlessRender</Configurations>
   </PropertyGroup>
Index: CefGlue.BrowserProcess/CefGlue.BrowserProcess.csproj
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/CefGlue.BrowserProcess/CefGlue.BrowserProcess.csproj b/CefGlue.BrowserProcess/CefGlue.BrowserProcess.csproj
--- a/CefGlue.BrowserProcess/CefGlue.BrowserProcess.csproj	(revision ef20aa2d39eaa29b410300ff2ab2b5e7cef4593a)
+++ b/CefGlue.BrowserProcess/CefGlue.BrowserProcess.csproj	(date 1675332401572)
@@ -2,7 +2,6 @@
 
   <PropertyGroup>
     <OutputType>WinExe</OutputType>
-    <TargetFrameworks>net472;$(DotnetVersion)</TargetFrameworks>
     <AssemblyName>Xilium.CefGlue.BrowserProcess</AssemblyName>
     <RootNamespace>Xilium.CefGlue.BrowserProcess</RootNamespace>
     <Configurations>Debug;Release;Debug_WindowlessRender</Configurations>
@@ -21,7 +20,7 @@
     <PackageReference Include="System.Runtime.Loader" />
   </ItemGroup>
 
-  <Target Name="PublishApp" AfterTargets="AfterBuild" Condition="'$(_AssemblyTimestampBeforeCompile)' != '$(_AssemblyTimestampAfterCompile)' and $(TargetFramework.StartsWith('$(DotnetVersion)')) and '$(IsPublishing)' != 'True'">
+  <Target Name="PublishApp" AfterTargets="AfterBuild" Condition="'$(_AssemblyTimestampBeforeCompile)' != '$(_AssemblyTimestampAfterCompile)' and '$(IsPublishing)' != 'True'">
     <!-- Publish both platforms on Windows for packaging -->
     <Message Text="Publishing..." Importance="High" />
     <MSBuild Condition="$(IsWindows) == 'True'" Projects="CefGlue.BrowserProcess.csproj" Targets="Publish" Properties="Configuration=$(Configuration);TargetFramework=$(TargetFramework);IsPublishing=True;PublishTrimmed=True;RuntimeIdentifier=win-x64" />
Index: Directory.Packages.props
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Directory.Packages.props b/Directory.Packages.props
--- a/Directory.Packages.props	(revision ef20aa2d39eaa29b410300ff2ab2b5e7cef4593a)
+++ b/Directory.Packages.props	(date 1675327278657)
@@ -23,6 +23,7 @@
         <PackageVersion Include="Dotnet.Bundle" Version="0.9.13" />
         <PackageVersion Include="System.Text.Json" Version="6.0.1" />
         <PackageVersion Include="System.Runtime.Loader" Version="4.3.0" />
+        <PackageVersion Include="System.CodeDom" Version="7.0.0" />
         <PackageVersion Include="NLog" Version="4.6.2" />
 
         <PackageVersion Include="cef.redist.x64" Version="$(CefRedistVersion)" />

